src/main/kotlin/com/terraformation/backend/funder/db/PublishedReportsStore.kt

```diff
@@ -127,6 +127,12 @@ class PublishedReportsStore(
     val metricTypeField =
         metricTable.field(
             "type_id", SQLDataType.INTEGER.asConvertedDataType(MetricTypeConverter()))!!
+    val unitField =
+        if (metricTable == PROJECT_METRICS) {
+          metricTable.field("unit", String::class.java)
+        } else {
+          DSL.value(null as String?)
+        }
```

Comment:
Can this be `metricTable.field("unit", String::class.java) ?: DSL.value(null as String?)`?

Reply:
As opposed to checking for a specific table, I mean. Take the `unit` field if it exists, otherwise null.

Reply:
Indeed, much cleaner. Updated